src/templates/template.ts

```diff
@@ -1,27 +1,32 @@
 import pug from 'pug';
 import { join } from 'path';
+import juice = require('juice');
 import { Connection } from 'typeorm';
+import htmlToText = require('html-to-text');
 
 import { Message } from '../worker';
 import { getMigrationSummary } from '../query';
+import { Template } from './templateInterface';
 
 let template: string;
```

Comment:
why are we declaring here instead of declaring on assignment

Reply:
it being used in many places.